Index: libbs2b-3.1.0/src/Makefile.am
===================================================================
--- libbs2b-3.1.0.orig/src/Makefile.am	2009-03-20 10:13:53.000000000 -0400
+++ libbs2b-3.1.0/src/Makefile.am	2014-01-02 11:28:11.433565170 -0500
@@ -18,7 +18,10 @@
     $(bs2b_HEADERS)
 
 libbs2b_la_LDFLAGS = \
-	-lm -version-info 0:0:0
+	-version-info 0:0:0
+
+libbs2b_la_LIBADD = \
+	-lm
 
 libbs2b_la_SOURCES = \
 	bs2b.c \
Index: libbs2b-3.1.0/configure.ac
===================================================================
--- libbs2b-3.1.0.orig/configure.ac	2009-06-04 14:17:02.000000000 -0400
+++ libbs2b-3.1.0/configure.ac	2014-01-02 11:39:23.214145269 -0500
@@ -5,7 +5,7 @@
 AC_INIT([libbs2b], [3.1.0], [boris_mikhaylov@users.sourceforge.net])
 AC_CONFIG_AUX_DIR([build-aux])
 AM_INIT_AUTOMAKE([1.10.1 -Wall foreign subdir-objects
-                  dist-zip dist-bzip2 dist-lzma])
+                  dist-zip dist-bzip2])
 AC_CONFIG_SRCDIR([src/bs2b.h])
 
 # Checks for programs.
